src/core/structured-log.js

```diff
@@ -277,12 +277,15 @@
       invoke('TRACE', mt, arguments);
     };
 
-    self.information = function(messageTemplate) {
+    self.verbose = self.trace;
+    self.debug = self.trace;
+
+    self.info = function(messageTemplate) {
       var mt = Array.prototype.shift.call(arguments);
       invoke('INFORMATION', mt, arguments);
```

Comment:
I like the move to `info` and `warn`. Perhaps the constants should be renamed also, to `INFO` and `WARN`?